src/dict.c

```diff
@@ -1750,6 +1753,7 @@ int dictTest(int argc, char **argv, int flags) {
     int retval;
     dict *dict = dictCreate(&BenchmarkDictType);
     long count = 0;
+    unsigned long newDictSize, currentDictUsed, remainKeys;
```

Comment:
I remember that we usually use new_dict_size for variable names

Reply:
Yes, you are right. I have fixed it.